packages/terra-divider/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-divider",
+  "main": "lib/Divider.js",
+  "version": "0.0.0",
+  "description": "The terra-divider component is used to separate content on a layout.",
```

Comment:
update to " The divider component..." to match READMEs

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/9502685e392616e794c4bc9c13238a1512a210f7#diff-b81595cc197ae1ed76ccb9c8b80a298fL4